debug/org.eclipse.debug.tests/src/org/eclipse/debug/tests/launching/LaunchConfigurationTests.java

```diff
@@ -1387,8 +1387,8 @@ public void launchesTerminated(ILaunch[] launches) {
 	/**
 	 * Tests that a launch is properly registered for notifications before a
 	 * process is spawned and may already propagate a termination event.
-	 *
-	 * @see https://github.com/eclipse-platform/eclipse.platform/issues/598
+	 * <p>
+	 * Seee https://github.com/eclipse-platform/eclipse.platform/issues/598
```

Comment:
This is pretty wrong (not even counting the typo).
It should become @see <a href="url"> ISSUE</a>

Reply:
If I am not mistaken `@see X` is only valid if `X` is a reference to some Java element (type, method).
Do you mean this should be `@see <a href="ISSUE_LINK"/>`?